### PR TITLE
BLAZ-25058-Added Readme file

### PR DESCRIPTION
**Pull request by SumathiSumi**
Hi @Keerthivasan-Ramamoorthy ,

I have added readme file in diagramBuilder.

Regards,
sumathi U.

### CODE REVIEW

File: MAUI/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Essential Studio for Blazor - Diagram Builder
+
+The diagram allows you to create a project in the MAUI platform. In this example, we can create a simple diagram with shapes as well as layout samples. We can also interact with the diagram.
+
+## Deployment
+
+### Requirements to run the demo
+
+The samples requires the below requirements to run.
```
Comment (florenceliliana) [inline MAUI/README.md:9]: The samples require the below requirements to run.

File: MAUI/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+* [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+* [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+
+### Run
+
+* Clone this repository.
+* Open the project in Visual studio 2022.
```
Comment (florenceliliana) [inline MAUI/README.md:17]: Open the project in Visual Studio 2022.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Essential Studio for Blazor - Diagram Builder
+
+The Blazor diagram builder application is used to create diagrams such as FlowChart and MindMap diagrams. In diagram builder application have all supports of diagram,, such as Print, Export, and Drawing Tool support, are available in the Diagram Builder application. You can interact with the diagram or create your own using shapes.
```
Comment (florenceliliana) [inline README.md:3]: The Blazor diagram builder application is used to create diagrams such as FlowChart and MindMap diagrams. In the diagram builder application, all support diagrams, such as Print, Export, and Drawing Tool support, are available in the Diagram Builder application. You can interact with the diagram or create your own using shapes.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+The samples requires the below requirements to run.
+
+* [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+* [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+
+### Run
+
+* Clone this repository.
+* Open the project in Visual studio 2022.
```
Comment (florenceliliana) [inline README.md:17]: Open the project in Visual Studio 2022.

File: wasm-side/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Essential Studio for Blazor - Diagram Builder
+
+You can create the diagram using Blazor WebAssembly App. In this example, we can create a simple diagram with shapes as well as layout samples. We can also interact with the diagram.
+
+## Deployment
+
+### Requirements to run the demo
+
+The samples requires the below requirements to run.
```
Comment (florenceliliana) [inline wasm-side/README.md:9]: The samples require the following requirements to run.

File: wasm-side/README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+* [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+* [.NET 6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
+
+### Run
+
+* Clone this repository.
+* Open the project in Visual studio 2022.
```
Comment (florenceliliana) [inline wasm-side/README.md:17]: * Open the project in Visual Studio 2022.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

**Review by sarathkumarvenkatesan — APPROVED** [review]

**Pull request merged**